lightkurve/prf/prfmodel.py

```diff
@@ -108,6 +108,66 @@ def evaluate(self, center_col, center_row, flux=1., scale_col=1., scale_row=1.,
                                                  grid=False).reshape(self.shape)
         return self.prf_model
 
+    def gradient(self, center_col, center_row, flux=1., scale_col=1., scale_row=1.,
+                 rotation_angle=0.):
+        """
+        This function returns the gradient of the KeplerPRF model with
+        respect to center_col, center_row, flux, scale_col, scale_row,
+        and rotation_angle.
+
+        Parameters
+        ----------
+        center_col, center_row : float
+            Column and row coordinates of the center
+        flux : float
+            Total integrated flux of the PRF
+        scale_col, scale_row : float
+            Pixel scale in the column and row directions
```

Comment:
To avoid confusion with Kepler's absolute pixel scale (~4 arcsec per pixel), the docstring here should should probably say something like `Pixel scale stretch parameter, i.e. the numbers by which the PRF model needs to be multiplied in the column and row directions to account for focus changes.`